### ISSUE TITLE
Infinity recursion in hot/poll / [HMR] Nothing hot updated.

### ISSUE THREAD

**Issue by axules** [webpack/webpack#13876]
![image](https://user-images.githubusercontent.com/20991327/127158687-5e9905ae-fb12-4a66-8e0b-bda421030612.png)

https://github.com/webpack/webpack/blob/beb42c64f696584ef570d8f57df448ddd0ac7238/hot/poll.js#L15

it should be like
```
if (!updatedModules || updatedModules.length === 0) {
  if (fromUpdate) log("info", "[HMR] Update applied.");
  return;
}
```

Application can't start and you can see this in the console
![image](https://user-images.githubusercontent.com/20991327/127159577-390bdead-c602-41f0-b1f2-521ed8129f60.png)

**Comment by axules** [comment]
PR https://github.com/webpack/webpack/pull/13877

**Comment by stefan-toubia** [comment]
Solves #13857 as well.

**Comment by axules** [comment]
fixed in v5.48.0
https://github.com/webpack/webpack/releases/tag/v5.48.0
